editor/components/inserter/menu.js

```diff
@@ -280,15 +280,21 @@ export class InserterMenu extends Component {
 		return this.renderCategories( visibleBlocks );
 	}
 
-	interceptArrows( event ) {
+	onTabbableEvent( event ) {
+		// If a tab (Recent, Blocks, …) is focused, pressing the down arrow
+		// moves focus to the selected panel below.
+		if (
+			event.keyCode === keycodes.DOWN &&
+			document.activeElement.getAttribute( 'role' ) === 'tab'
+		) {
+			return 1;
```

Comment:
From just reviewing this piece, it's unclear to me what `1` and `0` means. Any way to make this clearer?

Reply:
Amended the commit, hopefully making it clearer.